spec/mailers/workspaces_mailer_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+RSpec.describe WorkspacesMailer, type: :mailer do
+  let!(:user) { create(:user) }
+  let!(:workspace) { create(:workspace, user: user) }
+  let!(:file) { WickedPdf.new.pdf_from_string('<h1>Hello There!</h1>') }
+  
+  describe "workspace mailer" do
+
+    it 'send details to email' do
+      ActiveJob::Base.queue_adapter = :test
+      expect {WorkspacesMailer.workspace_details(workspace, user, file).deliver_later}.to have_enqueued_job.on_queue('mailers')
+    end
+
+    it 'email is sent' do
+      expect {
+        perform_enqueued_jobs do
```

Comment:
why do you need to execute job in mailer spec? remove this code
test only mailer